<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObturationView</class>
 <widget class="QWidget" name="ObturationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ObturationView</string>
  </property>
  <widget class="QComboBox" name="combo_color">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="post_check">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Радикуларен щифт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цвят:</string>
   </property>
  </widget>
  <widget class="QLabel" name="materialLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Материал:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="materialEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="SurfaceSelector" name="surfaceSelector" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>301</width>
     <height>143</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">View/uiComponents/qt_derived/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SurfaceSelector</class>
   <extends>QWidget</extends>
   <header>View/uiComponents/qt_derived/surfaceselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
